src/components/Table/GridTable.tsx

```diff
@@ -350,7 +368,7 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
 
     // If nestedCards is set, we assume the top-level kind is a card, and so should add spacers between them
     visitRows(maybeSorted, !!nestedCards);
-    nestedCards && nestedCards.done();
+    nestedCards && nestedCards.done(maybeSorted[maybeSorted.length - 1], filteredRows);
```

Comment:
For Chrome row `key` support, passing a Row was needed to create the proper unique `key` props. For this case, we expect that the last parent is being closed.

Reply:
Per other comment, but if we just use `chromeRow-${i++}` for keys, we wouldn't need to pass the previous row here. Not a big deal.

Reply:
Yeah! I did the cleanup :)